_posts/2016-06-22-hello-beercraft.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Maku brewing and Solita's passion pale ale
+author: Rinorragi
+excerpt: Solita ordered passion pale ale from Maku brewing and got introduction to their craftmenship
+tags: 
+- Beercraft
+---
+Solita wanted to create a beer of their own. The process was much like a software project. First we tried to figure out what we wanted. We had internal voting for the taste and name for the beer and "Intohimo" passion pale ale won the contest. After knowing what we need we went and sent multiple requests for proposals to different breweries. Among the offers we selected Maku brewing and decided with them that the receipt needs to be tested first. Maku brewing created 20l patch of the receipt they invented for us and at 21.6.2016 we got an opportunity to taste the beer and give feedback until final brew. All started with a bus trip from Solita's office at Helsinki to Tuusula. We had a great time already in the bus in discussion of projects and expectations of the upcoming beer. 
+
+## Maku brewing
+
+[Maku brewing](http://www.makubrewing.com/) is 2014 established finnish craft beer brewery making great beers. Their IPA and Amber ale are both award winning beers and really much for my liking. 
```

Comment:
is 2014 established finnish -> is a 2014 established Finnish

really much for -> really much to